pkgs/development/tools/ytt/default.nix

```
@@ -0,0 +1,30 @@
+{ pkgs, lib, ... }:
+with pkgs; with lib; buildGoModule rec {
+  pname = "ytt";
+  version = "0.30.0";
+
+  goPackagePath = "github.com/vmware-tanzu/carvel-ytt";
+  src = fetchFromGitHub {
+    sha256 = "0v9wp15aj4r7wif8i897zwj3c6bg41b95kk7vi3a3bzin814qn6l";
+    rev = "v${version}";
+    repo = "carvel-ytt";
+    owner = "vmware-tanzu";
+  };
+  vendorSha256 = null;
+
+  # on v0.30.0 YAML marshalling tests fail without consequence for ytt itself
+  checkPhase = "";
+  preFixup = ''
+    rm -f $out/bin/ytt-lambda-website
```

```suggestion
    rm $out/bin/ytt-lambda-website
```

Otherwise we won't notice when something changes.